### ISSUE TITLE
HL7/FHIR integration

### ISSUE THREAD

**Issue by bwalsh** [ga4gh/g2p-team#2]
The DWG is seeking to line up with the HL7/FHIR specification for exchanging Ehealth records, and in particular, their Genomics component which provides a means of reporting variants with optional assessed associated medical condition. I think this component bears most relevance here and so I am sharing the link below for the current interest of the G2P group and also for future reference as our APIs begin to align.

http://www.hl7.org/FHIR/2015May/observation-genetics-cg-prf-1a.html

Is this something that is applicable for our group? Please comment/vote below.

**Comment by bwalsh** [comment]
Initial examination of their documentation was interesting.  The browsable nature and easy access to sample data directly from the browser was productive.

One interesting set of typed attributes in their schema for identifiers.
This contrasts with GA4GH::Schema's use of string array 

```
  /** Names for the variant, for example a RefSNP ID. */
  array<string> names = [];

```

![image](https://cloud.githubusercontent.com/assets/47808/8834907/1eeedc56-306c-11e5-8bbc-53fead38c254.png)

``` json
 {
      "url": "http://hl7.org/fhir/StructureDefinition/geneticsDNASequenceVariationType",
      "valueCodeableConcept": {
        "coding": [
          {
            "system": "http://loinc.org",
            "code": "LA6690-7",
            "display": "Substitution"
          }
        ]
      }
    },
    {
      "url": "http://hl7.org/fhir/StructureDefinition/geneticsProteinReferenceSequenceId",
      "valueCode": "ENSESTP00000085772.1"
    },
    {
      "url": "http://hl7.org/fhir/StructureDefinition/geneticsAminoAcidChange",
      "valueString": "p.Leu858Arg"
    },
    {
      "url": "http://hl7.org/fhir/StructureDefinition/geneticsAlleleName",
      "valueString": "p.L858R"
    },
```

**Comment by GilAlterovitz** [comment]
Yes, this seems directly applicable- so G2P efforts could be used in clinical settings as well.

**Comment by bwalsh** [comment]
discussed October 1. decision to keep it in backlog.   desire to keep the `P` side of G2P robust.
